app/src/main/java/org/mozilla/fenix/ext/Activity.kt

```diff
@@ -48,3 +56,61 @@ fun Activity.breadcrumb(
         )
     )
 }
+
+/**
+ * Opens the set default browser setting depending on the user OS version:
+ * for devices with Q or lower takes the user to the settings page
+ * and for other allow the user to set the default browser in app.
```

Comment:
I don't think the 3 conditions are quite right from what I am reading in the code. You mean Android Q+ right for the first point?
In the second point, I think that applies to Android N+, but the else condition actually takes ppl to a sumo page. Provided some suggestion but I think we can also change this kDoc to say:
"Opens Android's Manage Default Apps Settings if possible."

Anyways, I want to be crystal that we are navigating out of the app into "Android's Manage Default Apps Settings".

```suggestion
 * - For Android Q+, navigate the user to Android's Manage Default Apps Settings
 * - For anything else, allow the user to set the default browser in app. (I think this is wrong??)
```

Reply:
Thanks for catching this, I will go with the straightforward documentation as you suggested "**Opens Android's Manage Default Apps Settings if possible."**